front/src/actions/message.js

```diff
@@ -96,7 +102,7 @@ function createActions(store) {
         actions.scrollToBottom();
         // then we send the message
         const createdMessage = await state.httpClient.post('/api/v1/message', newMessage);
-        const messagesWithoutTempMessage = store.getState().messages.filter(message => message.tempId !== tempId);
+        const messagesWithoutTempMessage = store.getState().messages.filter(message => message.id !== id);
```

Comment:
I'm not sure this is useful anymore, now that the ID is generated client-side? what's the purpose here?

Reply:
Deleted